docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -77,14 +77,14 @@ later).
 
         "module_stdout": "/bin/sh: /usr/bin/python: No such file or directory\r\n"
 
-     you can either set the :ref:`ansible_python_interpreter` inventory variable (see
+     you can either set the :ref:`ansible_python_interpreter<ansible_python_interpreter>` inventory variable (see
```

Comment:
We don't need to use the full angle bracket syntax when the displayed label and target are exactly the same.

Reply:
Occasionally the version without the angle brackets triggers a warning, and these were. I only added the angle brackets where I couldn't eliminate the errors any other way. I'll try it once more, though, before I give up and put the duplication in the code.